### ISSUE TITLE
Fix docstring errors in expanded_weights_utils.py, linear_fused.py, prepare.py, serializer.py, utils.py, batchnorm.py, functional_modules.py, conv_utils.py, linear_relu.py, activation.py, conv.py, normalization.py, dropout.py, linear.py, functional.py, convert_parameters.py, __init__.py, embedding_ops.py, conv_fused.py, sparse.py, rnn.py

### ISSUE THREAD

**Issue by svekars** [pytorch/pytorch#112632]
Please fix the following issues. To test locally, run: 
 ```
pydocstyle path-to-file --count
```
Run this command before and after fixing the errors. This command prints out the number of errors at the end of the output. Paste the output to your PR description with the number before and after. 
- **File**: `torch/nn/utils/convert_parameters.py`, **Entity**: `parameters_to_vector`, **Line**: 6, **Description**: First line should end with a period (not 'r')
- **File**: `torch/nn/utils/convert_parameters.py`, **Entity**: `vector_to_parameters`, **Line**: 28, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/utils/convert_parameters.py`, **Entity**: `_check_param_device`, **Line**: 58, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/utils/convert_parameters.py`, **Entity**: `_check_param_device`, **Line**: 58, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/utils/convert_parameters.py`, **Entity**: `_check_param_device`, **Line**: 58, **Description**: First line should end with a period (not 'd')
- **File**: `torch/nn/utils/convert_parameters.py`, **Entity**: `_check_param_device`, **Line**: 58, **Description**: First line should be in imperative mood; try rephrasing (found 'This')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `PackedSequence`, **Line**: 28, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `to`, **Line**: 119, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `to`, **Line**: 119, **Description**: First line should be in imperative mood (perhaps 'Perform', not 'Performs')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `is_cuda`, **Line**: 146, **Description**: First line should end with a period (not 'u')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `is_pinned`, **Line**: 150, **Description**: First line should end with a period (not 'y')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `is_pinned`, **Line**: 150, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `pad_packed_sequence`, **Line**: 274, **Description**: First line should be in imperative mood (perhaps 'Pad', not 'Pads')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `pad_sequence`, **Line**: 347, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `pad_sequence`, **Line**: 347, **Description**: First line should end with a period (not '`')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `unpad_sequence`, **Line**: 408, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `unpad_sequence`, **Line**: 408, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `pack_sequence`, **Line**: 454, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `unpack_sequence`, **Line**: 490, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/utils/rnn.py`, **Entity**: `unpack_sequence`, **Line**: 490, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `standard_kwargs`, **Line**: 19, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `standard_kwargs`, **Line**: 19, **Description**: Use """triple double quotes""" (found '''-quotes)
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `standard_kwargs`, **Line**: 19, **Description**: First line should end with a period (not 'e')
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `forward_helper`, **Line**: 28, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `forward_helper`, **Line**: 28, **Description**: Use """triple double quotes""" (found '''-quotes)
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `forward_helper`, **Line**: 28, **Description**: First line should end with a period (not ')')
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `sum_over_all_but_batch_and_last_n`, **Line**: 123, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `sum_over_all_but_batch_and_last_n`, **Line**: 123, **Description**: First line should end with a period (not 't')
- **File**: `torch/nn/utils/_expanded_weights/expanded_weights_utils.py`, **Entity**: `sum_over_all_but_batch_and_last_n`, **Line**: 123, **Description**: First line should be in imperative mood (perhaps 'Calculate', not 'Calculates')
- **File**: `torch/nn/utils/_expanded_weights/conv_utils.py`, **Entity**: `unfold3d`, **Line**: 189, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/nn/utils/_expanded_weights/conv_utils.py`, **Entity**: `unfold3d`, **Line**: 189, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/nn/utils/_expanded_weights/conv_utils.py`, **Entity**: `unfold3d`, **Line**: 189, **Description**: First line should be in imperative mood (perhaps 'Extract', not 'Extracts')
- **File**: `torch/nn/quantized/functional.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not 'l')
- **File**: `torch/nn/quantized/modules/activation.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/batchnorm.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/conv.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/dropout.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/embedding_ops.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/functional_modules.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/linear.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/normalization.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/rnn.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/__init__.py`, **Entity**: ``, **Line**: 1, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/modules/utils.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/dynamic/modules/__init__.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/dynamic/modules/conv.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/dynamic/modules/linear.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/dynamic/modules/rnn.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/_reference/modules/__init__.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/_reference/modules/conv.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/_reference/modules/linear.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/_reference/modules/rnn.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/_reference/modules/sparse.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantized/_reference/modules/utils.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/__init__.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/modules/__init__.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/modules/conv.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/modules/embedding_ops.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/modules/linear.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/dynamic/__init__.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/qat/dynamic/modules/linear.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/intrinsic/qat/modules/conv_fused.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/intrinsic/qat/modules/linear_fused.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/intrinsic/qat/modules/linear_relu.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantizable/modules/activation.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/nn/quantizable/modules/rnn.py`, **Entity**: ``, **Line**: 2, **Description**: First line should end with a period (not 's')
- **File**: `torch/backends/cudnn/__init__.py`, **Entity**: `version`, **Line**: 69, **Description**: First line should end with a period (not 'N')
- **File**: `torch/backends/cudnn/__init__.py`, **Entity**: `version`, **Line**: 69, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/backends/cudnn/__init__.py`, **Entity**: `is_available`, **Line**: 83, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/backends/mkl/__init__.py`, **Entity**: `is_available`, **Line**: 4, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/backends/mkl/__init__.py`, **Entity**: `verbose`, **Line**: 10, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/backends/mkl/__init__.py`, **Entity**: `verbose`, **Line**: 10, **Description**: First line should end with a period (not 'y')
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `is_built`, **Line**: 14, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `is_built`, **Line**: 14, **Description**: Multi-line docstring closing quotes should be on a separate line
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `is_built`, **Line**: 14, **Description**: First line should end with a period (not 's')
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `is_built`, **Line**: 14, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `cuFFTPlanCache`, **Line**: 38, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `cuFFTPlanCache`, **Line**: 38, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `cuFFTPlanCache`, **Line**: 38, **Description**: First line should end with a period (not 'e')
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `cuFFTPlanCacheManager`, **Line**: 59, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `cuFFTPlanCacheManager`, **Line**: 59, **Description**: First line should end with a period (not ',')
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `preferred_linalg_library`, **Line**: 121, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `preferred_linalg_library`, **Line**: 121, **Description**: Use """triple double quotes""" (found '''-quotes)
- **File**: `torch/backends/cuda/__init__.py`, **Entity**: `SDPBackend`, **Line**: 173, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/backends/openmp/__init__.py`, **Entity**: `is_available`, **Line**: 5, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/backends/mkldnn/__init__.py`, **Entity**: `is_available`, **Line**: 7, **Description**: First line should be in imperative mood (perhaps 'Return', not 'Returns')
- **File**: `torch/backends/mkldnn/__init__.py`, **Entity**: `verbose`, **Line**: 14, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/backends/mkldnn/__init__.py`, **Entity**: `verbose`, **Line**: 14, **Description**: First line should end with a period (not 'y')
- **File**: `torch/backends/_nnapi/prepare.py`, **Entity**: `NnapiInterfaceWrapper`, **Line**: 111, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/backends/_nnapi/prepare.py`, **Entity**: `ShapeComputeModule`, **Line**: 151, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/backends/_nnapi/prepare.py`, **Entity**: `ShapeComputeModule`, **Line**: 151, **Description**: First line should end with a period (not 'n')
- **File**: `torch/backends/_nnapi/serializer.py`, **Entity**: `ConvPoolArgs2d`, **Line**: 184, **Description**: 1 blank line required after class docstring (found 0)
- **File**: `torch/backends/_nnapi/serializer.py`, **Entity**: `_do_add_binary`, **Line**: 1294, **Description**: First line should end with a period (not 's')
- **File**: `torch/backends/_nnapi/serializer.py`, **Entity**: `_do_add_binary`, **Line**: 1294, **Description**: First line should be in imperative mood; try rephrasing (found 'Helper')
- **File**: `torch/backends/_nnapi/serializer.py`, **Entity**: `serialize_model`, **Line**: 2081, **Description**: No blank lines allowed after function docstring (found 1)
- **File**: `torch/backends/_nnapi/serializer.py`, **Entity**: `serialize_model`, **Line**: 2081, **Description**: 1 blank line required between summary line and description (found 0)
- **File**: `torch/backends/_nnapi/serializer.py`, **Entity**: `serialize_model`, **Line**: 2081, **Description**: First line should end with a period (not '

cc @carljparker
